content/blog/2023/web-pwa-native-RWA1.md

```diff
@@ -0,0 +1,136 @@
+  affecting project scope, timeline, and budget.
+  But understanding the differences between
+  a responsive web app,
+  a progressive web app,
+  or a native mobile app,
+  and which one is right for your project
+  can seem insurmountably difficult.
+  Welcome to part 1 of a three-part series
+  unpacking the reasons to choose
+  one platform over another.
+  Let's start with the responsive web app (RWA).
+---
+{% import 'embed.macros.njk' as embed %}
+
+You have an idea for a custom digital product you’d like to build, and it’s time to decide how to build it. One of your friends swears you won’t succeed unless you build a native mobile app for iPhone. A colleague argues that a responsive web app is the only way to go. Who’s right? Are there alternative options? The decision of how to build your app is quite important – affecting project scope, timeline, and budget – but understanding when to choose one option over another can seem insurmountably difficult.
+
+The great news is that you don’t need to spend years becoming a web developer yourself in order to make a wise choice. In the following three-part series, we’ll dig into the three main options for building mobile apps: Responsive Web Apps (RWAs), Progressive Web Apps (PWAs), and Native Mobile Apps, and how to determine the best option for your project. As you read, think about who your app will serve, what their most pressing needs are, and how your app will meet those needs.
+
+## What are RWAs, PWAs, and Native Mobile Apps?
+Here are the biggest differences in a nutshell.
+
+- **Responsive Web App (RWA)** - An application that runs in a browser, such as Chrome, Firefox, or Safari, and responds to fit any screen size on any device.
+- **Progressive Web App (PWA)** - A responsive web app with additional capabilities that mimic a native mobile app.
+- **Native Mobile App** - An app built specifically for one particular mobile operating system, like iOS or Android.
+
+
+{{ embed.img(
+  src='blog/2023/web-pwa-native-comparison-chart.png',
+  alt='A comparison chart showing the differences between
+  responsive web apps, progressive web apps, and native
+  mobile apps'
+) }}
+
+
+## Responsive Web Apps - Best for Wide Reach
+
+> **TL;DR**
+>
+> If your top priority is to reach as wide a range of
+> people as possible, as quickly as possible, while keeping
+> development and maintenance costs down, a responsive web app is
+> likely the best option for building your digital product.
+
+An RWA is essentially a website that resizes and rearranges its content – responding to fit whatever screen size a person chooses to use. RWAs are built using standard web technologies like HTML, CSS, and JavaScript. They are accessible from any device with a web browser such as Chrome, Firefox, Microsoft Edge, Safari, or a mobile browser.
+
+<div class="contain">
+
+{{ embed.img(
+  src='blog/2023/yes.png',
+  alt='The word yes with a smiley and sparkles',
+  attrs={
+    'class': 'align-left'
+  }
+) }}
```

Comment:
@SondraE I think you are having troubles with the align-left because on the image here because of the anchor link container that comes with the h2 heading levels. 

What we could do instead is assign a style to content-start in the attrs in order to have the image start at that wider grid column ... you'd have to remove the contain div (start and end)  but we can't control it becoming so small unless we move that to a CSS class (that doesn't exist yet) and edit what this class would do at different break points. 

See the screen recording and let me know what you think @SondraE  

https://user-images.githubusercontent.com/1581694/236315678-d9bc9554-7a5c-4363-a29a-4b1435f7e469.mov

Reply:
oh, and we'd have to change it to embed.figure

```

{{ embed.figure(
  data=[{
    img: 'blog/2023/yes.png',
    alt: 'The word yes with a smiley and sparkles'
  }],
  attrs={
    'style': '--column: content-start'
  }
) }}

```

Reply:
I added a css image helper code snippet that needs thought/work in another card: https://trello.com/c/f61Si0tU/214-remove-or-fix-unused-css-classes Not sure if we want to wait for that though.

Reply:
I don't think we should publish this as-is without either removing the yes/no images or fixing this.